Lib/javascript/v8/javascriptrun.swg

```diff
@@ -250,13 +256,47 @@ public:
 #else
   void (*dtor) (const v8::WeakCallbackInfo<SWIGV8_Proxy> &data);
 #endif
+
+  SWIGV8_ClientData() : index(-1) {}
+#if (V8_MAJOR_VERSION-0) < 4 && (SWIG_V8_VERSION < 0x031903)
+  v8::Persistent<v8::FunctionTemplate> class_templ(v8::Isolate* isolate,
+                                                   v8::Persistent<v8::FunctionTemplate>* v8_swig_types = NULL) {
+    return clazz_templ;
+  }
+#else
+  SWIGV8_FUNCTION_TEMPLATE class_templ(v8::Isolate* isolate,
+                                       v8::Persistent<v8::FunctionTemplate>* v8_swig_types = NULL) {
+#if (V8_MAJOR_VERSION-0) >= 4
+    if (v8_swig_types != NULL)
+      return SWIGV8_FUNCTION_TEMPLATE::New(isolate, v8_swig_types[index+1]);
+#endif
+    return SWIGV8_FUNCTION_TEMPLATE::New(isolate, clazz_templ);
+  }
+#endif
+  bool IsEmpty(v8::Persistent<v8::FunctionTemplate>* v8_swig_types = NULL) {
+#if (V8_MAJOR_VERSION-0) >= 4
+    if (v8_swig_types != NULL)
+      return v8_swig_types[index+1].IsEmpty();
+#endif
+    return clazz_templ.IsEmpty();
+  }
+  void Set(SWIGV8_FUNCTION_TEMPLATE class_templ, v8::Isolate *isolate,
+           v8::Persistent<v8::FunctionTemplate>* v8_swig_types = NULL) {
+#if (V8_MAJOR_VERSION-0) >= 4
+    if (v8_swig_types != NULL) {
+      v8_swig_types[index+1].Reset(isolate, class_templ);
+      return;
+    }
+#endif
+    if (clazz_templ.IsEmpty()) {
+      SWIGV8_SET_CLASS_TEMPL(clazz_templ, class_templ);
+    }
+  }
```

Comment:
Perhaps some consistency in the getter and setter naming would be better here... `GetClassTemplate()` and `SetClassTemplate()` ??

Reply:
Yes, sure. Just in case, the original rationale behind current naming choice is to make modification of callers minor. For example original `->class_templ` becomes `->class_templ()`. It's easier to review, but of course one can use the opportunity and make naming more self-consistent...